Assignments/Program.cs

```diff
@@ -1,9 +1,10 @@
 ﻿// ---------------------------------------------------------------------------------------
-// T13 - Strong Password
+// T15 - Isogram

```

Comment:
![image](https://github.com/vikrampg1710-tmm/Training/assets/75829713/9c5027cd-5a75-42f6-9a44-bbd57dd4d4f7)

If you want to follow the above format please redo this so that it has the copyright as well as the file details. Otherwise simply remove it.